docs/design/StratisSoftwareDesign.lyx

```diff
@@ -3357,7 +3357,7 @@ size
 \begin_inset Text
 
 \begin_layout Plain Layout
-number
+integer
```

Comment:
The issue here is that JSON at least according to https://en.wikipedia.org/wiki/JSON doesn't distinguish between integers and floating point and just has a `number` type. Of course we want to limit valid values not just to integers but usually (always?) to nonnegative integers. Saying integer is more accurate, but it does mean we're not strictly using JSON types any more.